rest-microservices-cart/src/test/resources/features/addProductInPortfolio.feature

```diff
@@ -59,3 +59,5 @@ Feature: Adding the product in the right risk bucket
       | ISIN357     | LOW        | country (with override) =  low > volatility =  high       |
       | ISIN486     | LOW        | country (with override) =  low > volatility =  medium     |
       | ISIN426     | LOW        | country (with override) and volatility = low -> low       |
+
+    #Are you sure | ISIN852     | MEDIUM     | country =  low < volatility =  medium                     |
```

Comment:
for ISIN852, country rating is 4, so risk is LOW. but volatility is D, which maps to MEDIUM. There's no override for Japan, so it should be MEDIUM